api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -284,7 +284,9 @@ message CommonResponse {
 
   // Instructions on how to manipulate the headers. When responding to an
   // HttpBody request, header mutations will only take effect if
-  // the current processing mode for the body is BUFFERED.
+  // the current processing mode for the body is: 1) BUFFERED; 2) or STREAMED and
+  // the below filter configuration is enabled:
+  // [#comment:TODO :ref:`continue_body_streaming_without_header_resp <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.continue_body_streaming_without_header_resp>]`
```

Comment:
Please put all the new additions in a comment tag.
See https://storage.googleapis.com/envoy-pr/a0691f1/docs/api-v3/service/ext_proc/v3/external_processor.proto.html#service-ext-proc-v3-commonresponse for how the docs look at the moment.

Reply:
ditto